lib/async_hooks.js

```diff
@@ -71,8 +71,7 @@ function fatalError(e) {
     Error.captureStackTrace(o, fatalError);
     process._rawDebug(o.stack);
   }
-  if (process.execArgv.some(
-      (e) => /^--abort[_-]on[_-]uncaught[_-]exception$/.test(e))) {
+  if (process.execArgv.some((e) => /^--abort[_-]on[_-]uncaught[_-]exception$/.test(e))) {
```

Comment:
I suppose one way to keep it readable and under 80 chars is to assign the regular expression to a variable on the previous line and use the variable here. I would prefer to avoid that because it means this PR is more than formatting changes. I'm not sure how reasonable that is of me, though. :-D